test/Mvc6Framework45.FunctionalTests/Models/IdentityModels.cs

```diff
@@ -24,7 +24,8 @@ public ApplicationDbContext()
             // Create the database and schema if it doesn't exist
             if (!_created)
             {
-                Database.AsRelational().ApplyMigrations();
+                Database.ApplyMigrations();
+                //Database.AsRelational().ApplyMigrations();
```

Comment:
I assume the Entity Framework API changed. If that's the case, why do we need the commented out old version here?